take-homes/fe-take-home.md

```diff
@@ -0,0 +1,29 @@
+# NY Times News Reader Take Home Test
+
+You will create a news reader application that pulls articles from the New York Times API.
+
+## Requirements:
+
+At a minimum, you must show:
+
+* A list of articles 
+* A "detailed" view for each article (as opposed to summary/list view)
+* Articles in the list must link to the detailed article view you create
+* Some sort of navigation, filter, OR sort
+
+Beyond the base requirements, include any additional elements you'd like, just be mindful of your time. You don't need to go overboard, just consider what might be necessary to achieve MVP so that this news feed is easy-to-use, information is clear, and navigation is painless. Put yourself in a user's shoes and think about what you would need (or want). We aren't looking for UI design, but we would like to see some demonstration of CSS knowledge, and the app should be usable on desktop, tablet, and mobile devices.
+
+## Things we'll be looking for:
+
+* a strong understanding of front-end development fundamentals (primarily JavaScript, but also CSS and HTML)
+* an understanding of React js best practices
+* an understanding of basic usability practices and standards
+* your ability to prioritize for MVP
+* a clear hierarchy of information
+* clean, well thought out code
+
+Try to limit your total time to around 6 hours on this project. Prioritize what is important to get this feed working and as close to MVP as possible. Please do not attempt to recreate NYTimes.com. We're interested in seeing your ability to write clean code to solve a problem with an understanding of basic UX. We're not looking for fancy UI tricks or hacks.
+
+## NYTimes API Info:
+
+Please use the top stories api located at https://developer.nytimes.com/docs/top-stories-product/1/overview
```

Comment:
Not as important, but maybe this bulletpoint about the API could get moved into the requirements next to `A list of articles` from....the [the top stories api])(link).

Reply:
I like it